### DISCUSSION TITLE
Check limit for free plan

### DISCUSSION THREAD

**Discussion by msafdev** [vercel/next.js#49982] [Help] [answered]
### Summary

Want to deploy my web in Vercel (new to this) but got "the check limit has exceeded free plan" warning, will the limit refresh tomorrow or is it a monthly limit?

**Comment by devjiwonchoi** [comment, marked as answer]
Your free trial limit is not going to refresh unless you move on to non-trial plan. If you don't need to publish your website for profit usage, you can use the [hobby plan](https://vercel.com/docs/concepts/limits/overview).
Or else you won't be able to use trial plan anymore.

Ref: https://vercel.com/docs/concepts/payments-and-billing/free-trials
